cmd/kube-scheduler/app/server.go

```diff
@@ -339,3 +309,46 @@ func WithPlugin(name string, factory framework.PluginFactory) Option {
 		return registry.Register(name, factory)
 	}
 }
+
+// SetupCommand creates a completed config based on the command args and options
+func SetupCommand(cmd *cobra.Command, args []string, opts *options.Options) (*schedulerserverconfig.CompletedConfig, error) {
```

Comment:
don't export this.

Reply:
but `SetupCommand` can be used by `cmd/kube-scheduler/app/testing/testserver.go`, otherwise we need to create the scheduler manually in `cmd/kube-scheduler/app/testing/testserver.go`.

Reply:
makes sense